plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsConnectorSmokeTest.java

```diff
@@ -19,38 +19,25 @@
 import com.azure.storage.blob.BlobServiceClientBuilder;
 import com.azure.storage.blob.models.BlobItem;
 import com.azure.storage.blob.models.ListBlobsOptions;
-import com.google.common.collect.ImmutableMap;
 import com.google.common.io.ByteSource;
-import com.google.common.io.Resources;
 import com.google.common.reflect.ClassPath;
-import io.trino.plugin.deltalake.util.DockerizedDataLake;
-import io.trino.plugin.deltalake.util.TestingHadoop;
+import io.trino.plugin.hive.containers.HiveHadoop;
```

Comment:
Please implement both `createHiveMinioDataLake` & `createDeltaLakeQueryRunner` methods.

Reply:
I've implemented those methods, I could see the hive container launching now. Let me know if it works on your side.

Reply:
I'm facing the below exception now:
```
java.lang.RuntimeException: Got exception: java.io.IOException No FileSystem for scheme: abfs
```

Reply:
I missed the overridden Hadoop version. Can you retest? Don't have an Azure account at the moment.

Reply:
It works now, thanks!